cpp/oneapi/dal/backend/primitives/distance/squared_l2_distance_dpc.cpp

```diff
@@ -14,6 +14,8 @@
 * limitations under the License.
 *******************************************************************************/
 
+#include "oneapi/dal/detail/profiler.hpp"
```

Comment:
Why profiler task is not created in this block?

Reply:
Fixed